packages/Operators/src/Factory/DTK_PointCloudOperatorFactory.hpp

```diff
@@ -90,7 +90,8 @@ class PointCloudOperatorFactory
     enum PointCloudOperatorType
     {
 	SPLINE_INTERPOLATION,
-	MOVING_LEAST_SQUARE
+	MOVING_LEAST_SQUARE,
+        NODE_TO_NODE
     };
```

Comment:
This type of factory design I had seen in Trilinos and emulated here.

I agree a builder/factory pattern would be more maintainable. That would also let users register their own stuff way easier (a different basis type for example). I think we should make an issue to refactor our factory scheme.